### PR TITLE
i18n(pt-BR): Add `islands.md` translation

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- Translated content

#### Description

Add PT-BR `islands.md` page translation

### CODE REVIEW

File: src/pages/pt-br/concepts/islands.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Ilhas Astro
+description: "Ilhas Astro (ou Ilhas de Componente) é um padrão de arquitetura web em que Astro é pioneiro. O termo “Arquitetura em ilhas” foi criado pelo arquiteto frontend da Etsy, Katie Sylor-Miller, em 2019 e expandido pelo criador do Preact, Jason Miller."
+i18nReady: true
+setup: |
+  import IslandsDiagram from '~/components/IslandsDiagram.astro';
+---
+
+**Ilhas Astro** (ou Ilhas de Componente) é um padrão de arquitetura web em que Astro é pioneiro. A ideia de “arquitetura em ilhas” foi criada pelo arquiteto frontend da Etsy, [Katie Sylor-Miller](https://twitter.com/ksylor), em 2019 e expandido [nesta postagem](https://jasonformat.com/islands-architecture/) pelo criador do Preact, Jason Miller.
```
Comment (caioferrarezi) [inline src/pages/pt-br/concepts/islands.md:10]: I would change this a little bit.
```suggestion
**Ilhas Astro** (ou Ilhas de Componente) é um padrão de arquitetura web em que Astro é pioneiro. A ideia de “arquitetura em ilhas” foi criada em 2019 pelo arquiteto frontend da Etsy, [Katie Sylor-Miller](https://twitter.com/ksylor) e expandido [nesta postagem](https://jasonformat.com/islands-architecture/) pelo criador do Preact, Jason Miller.
```

Comment (Yan-Thomas) [reply src/pages/pt-br/concepts/islands.md:10]: I agree, thanks!

File: src/pages/pt-br/concepts/islands.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+---
+// Exemplo: Utilize um componente React dinâmico na página.
+import MeuComponenteReact from '../components/MeuComponenteReact.jsx';
+---
+<!-- Este componente agora é interativo na página! 
+     O resto do seu website continua estático e com zero JS. -->
+<MeuComponenteReact client:load />
+```
+
+Com Ilhas Astro, a maior parte do seu site continua como leve e puro HTML e CSS. No exemplo acima, você apenas adicionou uma única e isolada **ilha de interatividade** sem precisar mudar o resto da sua página.
```
Comment (caioferrarezi) [inline src/pages/pt-br/concepts/islands.md:56]: I think this make more sense as a translation for "remain"

```suggestion
Com Ilhas Astro, a maior parte do seu site continua com o leve e puro HTML e CSS. No exemplo acima, você apenas adicionou uma única e isolada **ilha de interatividade** sem precisar mudar o resto da sua página.
```

File: src/pages/pt-br/concepts/islands.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Quais são os benefícios das Ilhas?
+
+O mais óbvio benefício de se construir com Ilhas Astro é performance: a maior parte do seu website é convertido em rápido HTML estático e JavaScript é apenas carregado para os componentes individuais que o precisam. JavaScript é um dos assets mais lentos que você pode carregar por byte, então cada byte conta.
```
Comment (caioferrarezi) [inline src/pages/pt-br/concepts/islands.md:60]:
```suggestion
O mais óbvio benefício de se construir com Ilhas Astro é performance: a maior parte do seu website é convertido em HTML estático e rápido, carregado JavaScript apenas para os componentes individuais que o necessitam. JavaScript é um dos assets mais lentos que você pode carregar por byte, então cada byte conta.
```

File: src/pages/pt-br/concepts/islands.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Outro benefício é carregamento paralelo, Na ilustração de exemplo acima, a ilha de baixa prioridade "carrossel de imagens" não precisa bloquear a ilha de alta prioridade "cabeçalho". Os dois são carregados em paralelo e hidratados de forma isolada, o que significa que o cabeçalho se torna interativo imediatamente sem precisar esperar que o carrossel, mais pesado, desacelere o carregamento da página.
```
Comment (caioferrarezi) [inline src/pages/pt-br/concepts/islands.md:62]:
```suggestion
Outro benefício é carregamento paralelo, na ilustração de exemplo acima, a ilha de baixa prioridade "carrossel de imagens" não precisa bloquear a ilha de alta prioridade "cabeçalho". Os dois são carregados em paralelo e hidratados de forma isolada, o que significa que o cabeçalho se torna interativo imediatamente sem precisar esperar que o carrossel, mais pesado, desacelere o carregamento da página.
```

Comment (caioferrarezi) [reply src/pages/pt-br/concepts/islands.md:62]: Maybe thi suggestion would be bettet
```suggestion
Outro benefício é carregamento paralelo. Na ilustração de exemplo acima, a ilha de baixa prioridade "carrossel de imagens" não precisa bloquear a ilha de alta prioridade "cabeçalho". Os dois são carregados em paralelo e hidratados de forma isolada, o que significa que o cabeçalho se torna interativo imediatamente sem precisar esperar que o carrossel, mais pesado, desacelere o carregamento da página.
```

File: src/pages/pt-br/concepts/islands.md (hunk 3 above)
Comment (Yan-Thomas) [reply src/pages/pt-br/concepts/islands.md:60]: Thanks, found a small typo but I'm going with your suggestion!
```suggestion
O mais óbvio benefício de se construir com Ilhas Astro é performance: a maior parte do seu website é convertido em HTML estático e rápido, carregando JavaScript apenas para os componentes individuais que o necessitam. JavaScript é um dos assets mais lentos que você pode carregar por byte, então cada byte conta.
```

File: src/pages/pt-br/concepts/islands.md (hunk 4 above)
Comment (Yan-Thomas) [reply src/pages/pt-br/concepts/islands.md:62]: I'm going with the second suggestion, thank you!

**Comment by Yan-Thomas** [comment]
Thanks for the amazing review @caioferrarezi! 💜

**Pull request merged**